rero_ils/modules/acq_orders/jsonschemas/acq_orders/acq_order-v0.0.1.json

```diff
@@ -0,0 +1,235 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "Acquisition order",
+  "description": "JSON schema for acq_order.",
+  "additionalProperties": false,
+  "propertiesOrder": [
+    "library",
+    "vendor",
+    "order_number",
+    "order_type",
+    "order_status",
+    "currency",
+    "order_date",
+    "description"
+  ],
+  "required": [
+    "$schema",
+    "pid",
+    "vendor",
+    "library",
+    "order_number",
+    "order_type",
+    "order_status",
+    "currency"
+  ],
+  "properties": {
+    "$schema": {
+      "title": "Schema",
+      "description": "Schema to validate acq_order records against.",
+      "type": "string",
+      "default": "https://ils.rero.ch/schema/acq_orders/acq_order-v0.0.1.json",
+      "pattern": "^https://ils.rero.ch/schema/acq_orders/acq_order-v([0-9]+?\\.){3}json$"
+    },
+    "pid": {
+      "title": "AcqOrder ID",
+      "type": "string",
+      "minLength": 1
+    },
+    "order_number": {
+      "title": "Order number",
+      "type": "string",
+      "minLength": 3,
+      "form": {
+        "validation": {
+          "validators": {
+            "valueAlreadyExists": {
+              "term": "order_number"
+            }
+          },
+          "messages": {
+            "alreadyTakenMessage": "The order number is already taken"
+          }
+        }
+      }
+    },
+    "order_type": {
+      "title": "Order type",
+      "type": "string",
+      "enum": [
+        "serial",
+        "monograph",
+        "standing order",
+        "monographic set",
+        "planned order",
```

Comment:
or planned_order same for all: we do not use space in code.

Reply:
Done